spec/requests/distributions_requests_spec.rb

```diff
@@ -14,7 +14,6 @@
   let(:hashed_id) { CGI.escape(crypt.encrypt_and_sign(organization.id)) }
   before(:each) do
     allow(Rails.application).to receive(:secret_key_base).and_return(secret_key)
-    allow(DistributionPdf).to receive(:new).and_return(double("DistributionPdf", compute_and_render: "PDF"))
```

Comment:
I assume this was being mocked to save time/resources? Locally I'm not seeing a significant difference in how long this test file takes to run with or without it. I think it might make more sense to add this mocking to the contexts that actually need the mock rather than just doing it at the top level here? But then again I'm trying to prove that my new feature doesn't cause the pdf to blow up and the only way to do that is to actually render it, but maybe that's less important?

Reply:
If everything works and it doesn't slow it down noticeably, it's fine to unstub it.